UI/data/themes/Acri.qss

```diff
@@ -1,3 +1,9 @@
+
```

Comment:
Starting a file with a new line is kind of weird.

Reply:
This must be left over from when the first token couldn't be `OBSThemeMeta`. I'll remove this.

Reply:
Done.